pkg/oktaApiAuth/api.go

```diff
@@ -192,25 +192,48 @@ func (auth *OktaApiAuth) cancelAuth(stateToken string) {
 	_, _, _ = auth.oktaReq(http.MethodPost, "/authn/cancel", data)
 }
 
+func processJsonAnswer(apiRes []byte, count int, nbFactors int) (AuthResponse, error) {
```

Comment:
```suggestion
func processJSONAnswer(apiRes []byte, count int, nbFactors int) (AuthResponse, error) {
```

https://google.github.io/styleguide/go/decisions.html#initialisms